endpoints-support.md

```diff
@@ -492,6 +540,8 @@
 |<sub>/rest/users/{userName}</sub>                                                        | DELETE   | :black_square_button: | :black_square_button: |    |
 |<sub>/rest/users/{userName}/roles</sub>                                                  | POST     | :black_square_button: | :black_square_button: |    |
 |<sub>/rest/users/{userName}/roles</sub>                                                  | PUT      | :black_square_button: | :black_square_button: |    |
+|     **Version**                                                                                                                         |
+|<sub>/rest/version</sub>                                                                 | GET      | :black_square_button: | :black_square_button: |                      |
```

Comment:
Actually this endpoint it is supported in Java, even though no sample was created. 
It is being used in OneViewConnector.

Reply:
Updated